src/reader_util.cpp

```diff
@@ -379,6 +379,15 @@ std::string ReaderUtil::Normalize(const std::string &str) {
 	UErrorCode err = U_ZERO_ERROR;
 	std::string res;
 	const icu::Normalizer2* norm = icu::Normalizer2::getNFKCInstance(err);
+	if (!norm || U_FAILURE(err)) {
```

Comment:
I do not think the `nullptr` check is necessary here, since ICU should always set the error code.